website/content/en/pre-docs/AWS/constraints.md

```diff
@@ -0,0 +1,159 @@
+---
+title: "Provisioning Configuration"
+linkTitle: "Provisioning"
+weight: 10
+---
+
+The [Provisioner CRD]({{< ref "../reference/provisioner-crd.md" >}}) provides two sections for constraining nodes. 
+
+- [`spec.requirements`](../reference/provisioner-crd#specrequirements)
+  - Cloud Provider Agnostic 
+  - Kubernetes Well Known Labels
+  - This section includues generally applicable constraints (zone, instance type) that each cloud provider is expected to implement. 
+  - Reference the Providioner CRD for more information. 
+- [`spec.provider`](#specprovider)
+  - Cloud Provider Specific
+  - This section defines constraints that are unique to AWS, such as SecurityGroups.
+  - Defined below. 
+
+## spec.provider
+
+cloud provider specific constrains
+
+[Review these fields in the code.](https://github.com/awslabs/karpenter/blob/main/pkg/cloudprovider/aws/apis/v1alpha1/provider.go#L33)
+
+### InstanceProfile
+An `InstanceProfile` is a region specific EC2 resource that is comprised of a reference to a single global IAM role.

```

Comment:
I found this sentence to be a little confusing. Do you think we need to distinguish the scope of an InstanceProfile (regional) and an IAM role (global)? Can we say that an instance profile is a way to pass an IAM role to an EC2 instance?

Reply:
I personally think that distinction is noteworthy, but probabbly not in karpenter docs.

I revised and shortened.